meta-sokol-flex-distro/conf/distro/sokol-flex.conf

```diff
@@ -174,9 +174,6 @@ INHERIT += "deploy-license-manifest"
 # current configuration
 INHERIT += "flex-updates-check"
 
-# Check PDK license
```

Comment:
what will happen if we keep this and only remove the layer?

Reply:
I am asking because this is shared layer so this will be used for the BSP where we need to enable license

Reply:
No problems were observed with the INHERIT in place.  I'll drop the change.